src/entity/spatial/LinearRing.cpp

```diff
@@ -0,0 +1,59 @@
+#include <sempr/entity/spatial/LinearRing.hpp>
+#include <LinearRing_odb.h>
+
+namespace sempr { namespace entity {
+
+SEMPR_ENTITY_SOURCE(LinearRing)
+
+LinearRing::LinearRing() : LinearRing(new core::IDGen<LinearRing>())
+{
+}
+
+LinearRing::LinearRing(const core::IDGenBase* idgen)
+    : LineString(idgen)
+{
+    this->setDiscriminator<LinearRing>();
+    geometry_ = factory_->createLinearRing();
+}
+
+LinearRing::~LinearRing()
+{
+    //factory_->destroyGeometry(geometry_);
```

Comment:
In the `Geometry`-class I read:
```c++
 Geometry::~Geometry() 
 {
    //factory geometry delete by sub classes!
 }
```
But here the delete is commented out?

Reply:
There was an issue with double freeing of the ring and string.
It was solved in 294dc770511d58b37ef3152be9e9f3d066d69643.